src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj

```diff
@@ -28,9 +28,13 @@
   </Target>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0'">
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjectionPackage3_0Version)" />
```

Comment:
Under 3.0 these can be reduced that already included by the `<FrameworkReference Include="Microsoft.AspNetCore.App" />`